lib/cell.rb

```diff
@@ -1,5 +1,10 @@
 class Cell
+  attr_reader :coordinate
   def initialize(coordinate)
+    @coordinate = coordinate
+  end
 
+  def coordinate
```

Comment:
This method is not necessary since the attr_reader you created does the same thing. Can you please delete the method and if any questions please do ask